classes/auth.php

```diff
@@ -929,7 +929,7 @@ public function update_user_profile_fields(&$user, $attributes, $newuser = false
                                     $user->$field = strtolower($attributes[$attr][0]);
                                 } else {
                                     // Custom profile fields have the prefix profile_field_ and will be saved as profile field data.
-                                    $user->$field = $attributes[$attr][0];
+                                    $user->$field = implode(',', (array) $attributes[$attr]);
```

Comment:
Just sanity checking, if the field set in attributes[attr] was NULL, is it expected that the value stored is an empty string (as it would be with this particular change),  or NULL?

Reply:
I think an empty string is fine. Null isn't a special case/value for these fields.